frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/AccessControlTab/AccessControlTab.tsx

```diff
@@ -40,7 +57,20 @@ export const AccessControlTab = ({ org, app }: AccessControlTabProps): ReactNode
 
   const { mutate: updateAppMetadataMutation } = useAppMetadataMutation(org, app);
 
-  const handleChange = (newPartyTypes: string[], currentPartyTypesAllowed: PartyTypesAllowed) => {
+  const handleChange = (
```

Comment:
Can we call this something more descriptive? `handleAllowedPartyTypeChange` or something

Reply:
fixed